lua/lspconfig/configs/texlab.lua

```diff
@@ -52,9 +52,15 @@ end
 
 local function buf_cancel_build()
   local bufnr = vim.api.nvim_get_current_buf()
-  if not util.get_active_client_by_name(bufnr, 'texlab') then
+  local client = util.get_active_client_by_name(bufnr, 'texlab')
+  if not client then
     return vim.notify('Texlab client not found', vim.log.levels.ERROR)
   end
+  if vim.fn.has 'nvim-0.11' == 1 then
+    return client:exec_cmd({
+      command = 'texlab.cancelBuild',
```

Comment:
```suggestion
    return client:exec_cmd({
      title = 'cancel',
      command = 'texlab.cancelBuild',
```

Reply:
Does `exec_cmd` support a `title` argument? Based upon documentation this is not clear

Reply:
it's a require field.

![QQ_1731233778465](https://github.com/user-attachments/assets/0196321d-c0fe-422f-ba22-34efed1244eb)